resync/api/monitoring_dashboard.py

```
@@ -386,40 +386,60 @@ async def disconnect(self, websocket: WebSocket):
         async with self._lock:
             self._clients.discard(websocket)
 
+    async def _subscribe_to_broadcast(self):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_subscribe_to_broadcast`

(ANN202)